Module-4/Task3/UserTaskController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using Task3.DoNotChange;
+
+namespace Task3
+{
+    public class UserTaskController
+    {
+        private readonly UserTaskService _taskService;
+
+        public UserTaskController(UserTaskService taskService)
+        {
+            _taskService = taskService;
+        }
+
+        public bool AddTaskForUser(int userId, string description, IResponseModel model)
+        {
+            string message = GetMessageForModel(userId, description);
+            if (message != null)
+            {
+                model.AddAttribute("action_result", message);
+                return false;
+            }
+
+            return true;
+        }
+
+        private string GetMessageForModel(int userId, string description)
```

Comment:
Try to come up with a better architecture for this class please. I think this method either needs refactoring or is not needed at all.
Right now, judging by the name of the method and its arguments, it is unclear to me what it is supposed to do and why